File: Day01/Day1.py
count = 0
previous = 0

with open('input.txt', 'r') as f:
    for line in f:
        if previous == 0:
            previous = int(line)
        else:
            if int(line) > previous:
                count += 1
            previous = int(line)

print(count)

File: Day01/Day1.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	file, err := os.Open("input.txt")
	if err != nil {
		fmt.Println(err)
	}
	defer file.Close()

	scanner := bufio.NewScanner(file)
	count := 0
	previous := 0
	for scanner.Scan() {
		if previous == 0 {
			previous, _ = strconv.Atoi(scanner.Text())
		} else {
			currentValue, _ := strconv.Atoi(scanner.Text())
			if currentValue > previous {
				count++
			}
			previous = currentValue
		}
	}
	fmt.Println(count)
}
